static/src/javascripts/bootstraps/enhanced/media/main.js

```diff
@@ -16,6 +16,7 @@ define([
     'common/modules/commercial/build-page-targeting',
     'common/modules/commercial/commercial-features',
     'common/modules/component',
+    'common/modules/experiments/ab',
```

Comment:
there isn't one! I branched off the other PR and squashed the commits... missed this change!